testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oauth/RefreshTokenTest.java

```diff
@@ -121,6 +122,9 @@ public class RefreshTokenTest extends AbstractKeycloakTest {
     @Rule
     public AssertEvents events = new AssertEvents(this);
 
+    @Rule
+    public ReverseProxy proxy = new ReverseProxy();
```

Comment:
Is it possible to update this to `ClassRule` to avoid restart undertow server at every test method?